lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -673,6 +683,207 @@ func (c *ContainerProxy) StreamContainerStats(ctx context.Context, config *conve
 	return nil
 }
 
+// StatPath requests the portlayer to stat the filesystem resource at the
+// specified path in the container vc.
+func (c *ContainerProxy) StatPath(ctx context.Context, name, path string) (*types.ContainerPathStat, error) {
+	defer trace.End(trace.Begin(name))
+
+	vc := cache.ContainerCache().GetContainer(name)
```

Comment:
you should just pass vc in from the backend.  We have a bunch of functions in the proxy that takes vc as an arg.  You will still need name to get passed in as name can be friendly name or alphanumeric ID.